packages/nodes-base/nodes/ActiveCampaign/__schema__/v1.0.0/account/get.json

```diff
@@ -0,0 +1,110 @@
+{
```

Comment:
are we planning to auto-generate these schema files?

Reply:
Yes we have a workflow that will automatically make PRs with these based on telemetry data, example branch here:
https://github.com/n8n-io/n8n/tree/schema-generation-17842463